src/js/game.ts

```diff
@@ -806,6 +824,14 @@ export class Game {
     return !this.audioSources[audio].paused;
   }
 
+  spriteSpeak = (key: string) => {
```

Comment:
Write this in usually function style similar to how I converted the other functions as per styleguide.